### DISCUSSION TITLE
Add folder structures to collections and the Admin UI

### DISCUSSION THREAD

**Discussion by janus-reith** [payloadcms/payload#5664] [Feature Requests & Ideas]
Similar to my request to add [tree structures](https://github.com/payloadcms/payload/discussions/5663), it would be great to allow organizing documents in folders.

In contrast to the trees, folders would need additonal data, I wonder what the best way would be to schematize these.
Making them stub documents within the collection would align with the existing child->parent relation, but that's probaby not a good idea, especially since some collections define required fields.
It would probably make sense to store folders as a separate Entity

This might in some parts align with considerations for [tree structures](https://github.com/payloadcms/payload/discussions/5663), I however made folders a separate discussion since the former could probably be implemented without requiring changes to the data model.

**Comment by mysteren** [comment]
https://github.com/payloadcms/payload/discussions/1961#discussioncomment-11806159

**Comment by GermanJablo** [comment]
Closing as duplicated of https://github.com/payloadcms/payload/discussions/4190
This feature is a WIP and we are very close, so stay tuned! Thank you for your patience!
